### ISSUE TITLE
Client should prefer OPENLINEAGE_ variables over MARQUEZ_ ones

### ISSUE THREAD

**Issue by rossturk** [OpenLineage/OpenLineage#830]
The client currently checks for the existence of MARQUEZ_ environment variables and uses them, then falls back on OPENLINEAGE_ equivalents.

This should be reversed, to show preference for the newer variable names.

**Comment by Sheeri** [comment]
I can't see where this issue is in the code. there's nothing in the client/ directory that indicates this behavior is occurring. 

And it looks like it's already covered - specifically, in:
integration/airflow/openlineage/airflow/adapter.py
```
_DAG_NAMESPACE = os.getenv('OPENLINEAGE_NAMESPACE', None)
if not _DAG_NAMESPACE:
    _DAG_NAMESPACE = os.getenv(
        'MARQUEZ_NAMESPACE', _DAG_DEFAULT_NAMESPACE
    )
```

---

The only place that has any value of "marquez" (case-insensitive) in client is a test:

; find . -type f -exec grep -li marquez {} \;
./python/tests/test_client.py
;

and that's not even a variable:
    assert OpenLineageClient(url="  http://marquez:5000  ", session=session).transport.url == \
           "http://marquez:5000"
    assert OpenLineageClient(url="  https://marquez  ", session=session).transport.url == \
           "https://marquez"

The only thing I could find that matches is in integration/airflow/tests/test_openlineage_adapter.py - MARQUEZ comes before OPENLINEAGE there, though I'm not sure the order matters. If it does, I can submit a PR for that change.

**Comment by arumugamtp** [comment]
Hi,
Is this issue still open? Good for new person to play-around?

**Comment by mobuchowski** [comment]
@arumugamtp yes - although at this point it should be about removing Marquez ones
